### PR TITLE
Usable Item Additions and Refinements

### PR DESCRIPTION
**Pull request by neuromancerxi**
* Adds Scripts to make the following items usable:

Catholicon / +1
Clerics / Vicars Drink
Healing Mist / Powder
Mana Mist / Powder
Lucid Wings I / II
Sprinters Drink

* Migrates Effect Removal from individual Salve Scripts into item_utils
* Updates comments for a few item scripts
* Updates use time for a few items

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Makes a number of items usable.

## Steps to test these changes

Login in a Test Character, add temp items, verify effects.
```
4146 -- Revit
4202 -- Daed wing
4206 -- catholicon
4208 -- cath +1
4254 -- Megalix
4255 -- mana powder
5322 -- healing powder
5395 -- clerics drink
5397 -- sprinters
5439 -- vicars
5440 -- Dusty Wing
5832 -- healing mist
5833 -- mana mist
5834 -- lucid wings i
5837 -- clear salve i
5838 -- clear salve ii
6475 -- lucid wings ii
```

### CODE REVIEW

File: scripts/globals/item_utils.lua [hunk 1]
```diff
@@ -133,3 +134,29 @@ xi.item_utils.addTwoItemEffects = function(target, effect1, effect2, power1, pow
...
     end
 end
+
+xi.item_utils.removeMultipleEffects = function(target, effects, count, random)
+    if random == 1 then -- randomize which effects get removed
+        local effects = utils.shuffle(effects)
+    end
+
+    if count > 0 then
+        local function removeStatus()
```
Comment (claywar) [inline scripts/globals/item_utils.lua]: Would prefer to declare this local function outside of the containing function for visibility.  Should have no impact on your code.

File: scripts/globals/item_utils.lua [hunk 2]
```diff
@@ -133,3 +134,29 @@ xi.item_utils.addTwoItemEffects = function(target, effect1, effect2, power1, pow
...
+            for _, effect in ipairs(effects) do
+                if target:delStatusEffect(effect) then return true end
```
Comment (claywar) [inline scripts/globals/item_utils.lua]: Please break out conditions into multiple lines

File: scripts/globals/items/bottle_of_catholicon.lua [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+require("scripts/globals/item_utils")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local effects =
```
Comment (claywar) [inline scripts/globals/items/bottle_of_catholicon.lua]: Please declare this table outside of function, similar to above

File: scripts/globals/items/bottle_of_catholicon_+1.lua [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+require("scripts/globals/item_utils")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local effects =
```
Comment (claywar) [inline scripts/globals/items/bottle_of_catholicon_+1.lua]: Same as above

File: scripts/globals/items/bottle_of_clerics_drink.lua [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+require("scripts/globals/item_utils")
+-----------------------------------
+local item_object = {}
+
+item_object.onItemCheck = function(target)
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local effects =
```
Comment (claywar) [inline scripts/globals/items/bottle_of_clerics_drink.lua]: Since this pattern is repeated, it should most likely be declared elsewhere and utilized.

**Review by claywar — APPROVED** [review]

File: scripts/globals/item_utils.lua [hunk 6]
```diff
@@ -3,10 +3,48 @@
 -----------------------------------
 require("scripts/globals/status")
 require("scripts/globals/msg")
+require("scripts/globals/utils")
 -----------------------------------
 xi = xi or {}
 xi.item_utils = {}
 
+xi.item_utils.removableEffects =
```
Comment (zach2good) [inline scripts/globals/item_utils.lua:11]: 👍

**Review by zach2good — APPROVED** [review]

**Pull request merged**